src/server/locallib.php

```diff
@@ -18,7 +18,9 @@
 
 defined('MOODLE_INTERNAL') || die();
 
+# **** Which file does this line refer to? ****
```

Comment:
the vendor folder is generated when composer is run. It loads external libraries used by this project